maze_transformer/generation/lattice_maze.py

```diff
@@ -261,6 +315,10 @@ def from_adj_list(
             connection_list=connection_list,
         )
 
+    # ============================================================
+    # TODO: write a to_tokens method?
```

Comment:
Definitely! 😄 It can probably just call `maze_to_tokens`

Reply:
Or maybe we just move `maze_to_tokens` into this class as `to_tokens`? Not sure if we have any need for it being seperate.

Reply:
out of scope for this PR, leaving this for later